crates/bevy_app/src/sub_app.rs

```diff
@@ -332,6 +333,7 @@ impl SubApp {
     }
 
     /// See [`App::get_added_plugins`].
+    #[cfg(feature = "downcast")]
     pub fn get_added_plugins<T>(&self) -> Vec<&T>
```

Comment:
This is the only real consequence of losing `downcast-rs` support on certain `no_std` targets (not all!). I would like to bring feature parity regardless, but it is fairly minor IMO.